mods/cnc/maps/shellmap/map.yaml

```diff
@@ -994,6 +994,9 @@ Rules:
 			Effect: Desaturated
 		LuaScript:
 			Scripts: shellmap.lua
+		MusicPlaylist:
+			StartingMusic: map1
```

Comment:
Yeah, https://github.com/OpenRA/OpenRA/pull/8562/files#diff-0612b1f741b8612131b70560bfabc77aR57 is a check I carried over which causes it. I guess it should be removed there.